### ISSUE TITLE
$load by fhir endpoint doesn't translate to aidbox types

### ISSUE THREAD

**Issue by jyeros** [Aidbox/Issues#278]
When use load by fhir endpoint the reference isn't translate to aidbox format if use aidbox format by `/fhir/$load` it works. Here there is an example with only a patient
[patient.ndjson.gz](https://github.com/Aidbox/Issues/files/4330954/patient.ndjson.gz)

devbox; 0.4.9; bdf14e28;

**Comment by niquola** [comment]
Could not reproduce:

```

POST /fhir/Patient/$load

source: '/Users/niquola/Downloads/patient.ndjson.gz'

----

GET /Patient

....
entry:
- resource:
    managingOrganization: {id: org1, resourceType: Organization}
    id: pat1
    resourceType: Patient
    meta: {lastUpdated: '2020-03-16T10:13:25.433750Z', createdAt: '2020-03-16T10:13:25.433750Z', versionId: '85'}
```

**Comment by jyeros** [comment]
@niquola It works if specify the `resourceType` but if use global `$load` operation doesn't work, for example 
```
POST /fhir/Patient/$load

source: '/Users/niquola/Downloads/patient.ndjson.gz'
```
This works ☝️ 

and this doesn't work 👇 
```
POST /fhir/$load

source: '/Users/niquola/Downloads/patient.ndjson.gz'
```

**Comment by niquola** [comment]
Fixed in 0.5.0-SNAPSHOT
